### PR TITLE
DevEnv: update frontend dependencies - npm, nodemon, typescript

### PR DESCRIPTION
**Pull request by Estrax**
**What this PR does / why we need it**:

Updates frontend dependencies to the newest versions as of the day of submitting this PR (Feb 12 2020)

**Which issue(s) this PR fixes**:

Fixes #20574

**Special notes for your reviewer**:

Changes - more significant:
- `nodemon` - changes (1.18.10 -> 2.0.2): https://github.com/remy/nodemon/releases

Changes - less significant:
- `npm` - changes (6.13.4 -> 6.13.7): https://github.com/npm/cli/blob/latest/CHANGELOG.md
- `typescript` - changes (3.7.2 -> 3.7.5): https://github.com/microsoft/TypeScript/releases

### CODE REVIEW

**Review by dprokop — CHANGES_REQUESTED** [review]
yarn.lock seems broken

**Comment by Estrax** [comment]
@dprokop fixed!

**Review by hugohaggmark — DISMISSED** [review]
Still conflicts

File: package.json [hunk 1]
```diff
@@ -117,7 +117,7 @@
     "monaco-editor": "0.15.6",
     "ngtemplate-loader": "2.0.1",
     "node-sass": "4.13.1",
-    "npm": "6.13.4",
+    "npm": "6.13.7",
```
Comment (dprokop) [inline package.json]: Wat? I've never noticed npm in our dependencies. Is this necessary? My gut says we can remove it, unless @torkelo has some background details for introducing it a long time ago (https://github.com/grafana/grafana/blame/010a29a42dd347e382daaf361522d3862b725b95/package.json#L91)

Comment (torkelo) [reply package.json]: Yea, that is strange

Comment (Estrax) [reply package.json]: @dprokop sure; removed.

**Comment by dprokop** [comment]
Forgot about one thing-> when updating dependencies in core you also need to update those across packages. In this case it's typesctript update in grafana/ui, data, toolkit and runtime. Update in grafana/toolkit we need to communicate in changelog.

**Comment by Estrax** [comment]
@dprokop yep, that's in progress; gonna roll out updates in packages about the beginning of next week, just didn't wanted to open too many PRs at once. Updates of dependencies in packages are going to be in separate PRs to keep it easy to manage.

**Comment by dprokop** [comment]
why you want to keep them separate? I think packages updates should be cross-package as separating those two can lead to some unpredictable issues with packages versions

**Comment by hugohaggmark** [comment]
Any updates on this @Estrax?

**Review by dprokop — APPROVED** [review]
Seems to be quite safe to merge :)

**Pull request merged**